cuML/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ endif(NOT DISABLE_OPENMP OR NOT ${DISABLE_OPENMP})
 
 ###################################################################################################
 # - Submodules ------------------------------------------------------------------------------------
-
+set(CUDA_INCLUDE_DIR /usr/local/cuda/include CACHE STRING "CUDA Include dir")
```

Comment:
using `CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES` would work here, wouldn't it?

Reply:
> Biggest thing is the need to update build instructions

This is exactly why I'm glad we follow "Review Then Commit" contribution process. This was an oversight. I had a PR get merged in on the FAISS side that provides a "make install" option for the gpu artifact. 

In order for contributors / developers of cuML to build against FAISS, they will actually need to build and install FAISS from source such that the artifacts are provided. I will open a ticket for FAISS to build deb / rpm packages to make this easier, however, in the meantime you can build FAISS with the following flags to have it install into the standard /usr/lib64 and /usr/include locations (rather than /usr/local/lib64 and /usr/local/include). 

`--libdir /usr/lib64 --includedir /usr/include`

I'm in discussion w/ the Facebook team on changing the defaults.

Reply:
Is there no way to install the necessary artifacts are installed in the python/conda environment? As in we could install them to `$CONDA_PREFIX` when using conda so that the faiss files are pertinent to that environment only, no?